XamlControlsGallery/ControlPagesSampleCode/NumberBox/NumberBoxSample2_xaml.txt

```diff
@@ -0,0 +1,6 @@
+﻿<controls:NumberBox x:Name="NumberBoxSpinButtonPlacementExample"
+    Header="Enter an integer:" 
+    Value="1" 
+    SpinButtonPlacementMode="Compact" 
+    StepFrequency="1" >
+</controls:NumberBox>
```

Comment:
Thanks, @chingucoding. I was following the TeachingTip sample, but it indeed looks like we have changed to the "muxc" namespace for new controls. I'll file a bug for myself on the TeachingTip sample so that I get that updated as well.

Reply:
Fixed!